static/js/custom/tables/measurements-table.js

```diff
@@ -25,16 +25,40 @@ $(document).ready(function() {
                         targets: [7, 8, 9, 10, 11],
                         defaultContent: '',
                         render: function(data, type, row, meta) {
+                            
                             if (type == 'display') {
-                                if (!(mea_types === undefined) && !(data === null)) {
+                                if (!(mea_types === undefined) && !(data === null) && !(data === "None")) {
```

Comment:
The data shouldn't be "None", I think it's better if we just don't display the value in the table at all if it's None, as it's cleaner and would help users with visibility